lib/resources/teams/tags/test.js

```diff
@@ -0,0 +1,55 @@
+import LatitudeSh from '../../../latitudesh.js';
+
+const LatitudeShApi = new LatitudeSh('fake token');
+const headers = {
+  Authorization: 'Bearer fake token',
+  'Content-Type': 'application/json',
+};
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('get team tags', () => {
```

Comment:
Since we are testing post and delete as well, it would be nice to update this description.